### PR TITLE
Feature/documentation introduction

### PR DESCRIPTION
**Pull request by Quimnadalsalle**

### CODE REVIEW

File: doc/build/html/_sources/introduction.rst.txt [hunk 1]
```diff
@@ -0,0 +1,46 @@
+Introduction
+==========================
+
+**The Sheriff** is an exciting game application about an **outlaw** that
+goes out to eat with his **gang friends** in some restaurant. Along the game,
+the player will be able to **organize raids** in any restaurant, **create new gangs** and
+**many more features**!
+
+This game is created by the the company *StartupZ*, leaded by Fran Mangas,
+who brang the business project, and developed by *The Sheriff Team*, a group of Software
```
Comment (tbanjos) [inline doc/build/html/_sources/introduction.rst.txt]: small typo: bring

File: doc/build/html/_sources/index.rst.txt [hunk 2]
```diff
@@ -4,23 +4,51 @@
...
+===============================
+Missing explanation of why whe choose this architecture
+
+- The Sheriff Application
+
+Missing explanation of architectural decisions in Application layer
+
+- The Sheriff Domain
+
+Missing explanation of architectural decisions in Domain layer
```
Comment (tbanjos) [inline doc/build/html/_sources/index.rst.txt]: Here we put our domain classes (Outlaw, Gang, Raid) containing all pertinent business logic and the factory classes.

File: doc/build/html/_sources/index.rst.txt [hunk 3]
```diff
@@ -4,23 +4,51 @@
    contain the root `toctree` directive.
 
 Welcome to The Sheriff's documentation!
-=======================================
+***************************************
```
Comment (tripledes) [inline doc/build/html/_sources/index.rst.txt]: I'd say this should be consider the Document title and should be:

```
================
 Document Title
================
```

File: doc/build/html/_sources/index.rst.txt [hunk 4]
```diff
@@ -4,23 +4,51 @@
...
+Missing explanation of architectural decisions in Application layer
+
+- The Sheriff Domain
+
+Missing explanation of architectural decisions in Domain layer
+
+
+- The Sheriff Infraestructure
+
+Missing explanation of architectural decisions in Infraestructure layer
```
Comment (tbanjos) [inline doc/build/html/_sources/index.rst.txt]: Contains the REST API endpoints and MySQL repositories

**Comment by tripledes** [comment]
@Quimnadalsalle other than the two comments and the conflicts, looks good :+1:

File: doc/build/html/_sources/introduction.rst.txt [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+The goal of doing this project is to **learn**, **apply** and later **evaluate** what the *Evil Corp*
+developers learnt in *Diseño Software 1* subject:
+
+1. **Clean code technics**
+
+2. **Creational, structural and behavioral patterns**
+
+3. **SOLID principles.**
+
+**The Sheriff** is an activity is contained in *Diseño Software 1* subject, done in the first semester of the
```
Comment (tripledes) [inline doc/build/html/_sources/introduction.rst.txt]: **The Sheriff is an activity contained?

File: doc/build/html/_sources/introduction.rst.txt [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+*Master en Desarrollo y Arquitectura de Software* degree, in *La Salle, Universitat Ramón
+Llull* university.
+
+Document Contents
+-----------------
+
+In all the other sections basically it is going to be explained the next information:
+
+- **Functionality - The Sheriff usage**: How to run the application, how to run the code static analysis,
```
Comment (tripledes) [inline doc/build/html/_sources/introduction.rst.txt]: The document `usage.rst` was aiming exactly this.

File: doc/build/html/usage.html [hunk 7]
```diff
@@ -179,7 +182,7 @@ <h3 id="searchlabel">Quick search</h3>
       &copy;2019, The Sheriff Team.
       
       |
-      Powered by <a href="http://sphinx-doc.org/">Sphinx 2.2.0</a>
+      Powered by <a href="http://sphinx-doc.org/">Sphinx 2.3.1</a>
```
[the remark this answers is not in the document]
Comment (Quimnadalsalle) [reply doc/build/html/usage.html]: No, I didn't consider this. I will update the Pipfile. Sorry!

File: doc/build/html/_sources/index.rst.txt (hunk 3 above)
Comment (Quimnadalsalle) [reply doc/build/html/_sources/index.rst.txt]: See my answer in Slack

File: doc/build/html/_sources/introduction.rst.txt (hunk 6 above)
Comment (Quimnadalsalle) [reply doc/build/html/_sources/introduction.rst.txt]: I know, and the introduction introduces that we are going to talk about this later!

**Pull request merged**